pkgs/applications/editors/emacs/elisp-packages/manual-packages/cask/package.nix

```diff
@@ -1,67 +1,58 @@
-{ lib
-, stdenv
-, fetchFromGitHub
-, bash
-, emacs
-, python3
+{
+  lib,
+  bash,
+  emacs,
+  fetchFromGitHub,
+  melpaBuild,
+  python3,
 }:
 
-stdenv.mkDerivation (finalAttrs: {
-  pname = "cask";
+let
   version = "0.8.8";
+in
+melpaBuild {
+  pname = "cask";
+  inherit version;
 
   src = fetchFromGitHub {
     owner = "cask";
     repo = "cask";
-    rev = "v${finalAttrs.version}";
+    rev = "v${version}";
     hash = "sha256-TlReq5sLVJj+pXmJSnepKQkNEWVhnh30iq4egM1HJMU=";
   };
 
-  doCheck = true;
+  buildInputs =
+    [
+      bash
+      python3
+    ]
+    ++ (with emacs.pkgs; [
+      ansi
+      dash
+      ecukes
+      el-mock
+      ert-async
+      ert-runner
+      f
+      git
+      noflet
+      package-build
+      s
+      servant
+      shell-split-string
```

Comment:
Usually, elisp packages are directly listed as parameters at the top of this file and added as `packageRequires`.  Is there a reason to break the convention here?

Reply:
None besides a slip.